src/cmd.rs

```diff
@@ -1919,6 +1942,97 @@ command!(
     }
 );
 
+fn print_statefulsets(
+    list: StatefulSetList,
+    regex: Option<Regex>,
+    writer: &mut ClickWriter,
+) -> StatefulSetList {
+    let mut table = Table::new();
+    table.set_titles(row!["####", "Name", "Desired", "Current", "Ready"]);
+    let statefulsets_specs = list.items.into_iter().map(|statefulset| {
+        let mut specs = Vec::new();
+        specs.push(CellSpec::new_index());
+        specs.push(CellSpec::new_owned(
+            val_str("/metadata/name", &statefulset, "<none>").into_owned(),
+        ));
+        specs.push(CellSpec::new_owned(format!(
+            "{}",
+            val_u64("/spec/replicas", &statefulset, 0)
+        )));
+        specs.push(CellSpec::new_owned(format!(
+            "{}",
+            val_u64("/status/currentReplicas", &statefulset, 0)
+        )));
+        specs.push(CellSpec::new_owned(format!(
+            "{}",
+            val_u64("/status/readyReplicas", &statefulset, 0)
+        )));
+        (statefulset, specs)
+    });
+
+    let filtered = match regex {
+        Some(r) => ::table::filter(statefulsets_specs, r),
+        None => statefulsets_specs.collect(),
+    };
+
+    ::table::print_table(&mut table, &filtered, writer);
+
+    let final_statefulsets = filtered
+        .into_iter()
+        .map(|statefulset_spec| statefulset_spec.0)
+        .collect();
+
+    StatefulSetList { items: final_statefulsets }
+}
+
+command!(
+    StatefulSets,
+    "statefulsets",
+    "Get statefulsets (in current namespace if set)",
+    |clap: App<'static, 'static>| clap.arg(
+        Arg::with_name("show_label")
+            .short("L")
+            .long("labels")
+            .help("Show statefulsets labels")
+            .takes_value(true)
+    ).arg(
+        Arg::with_name("regex")
+            .short("r")
+            .long("regex")
+            .help("Filter statefulsets by the specified regex")
+            .takes_value(true)
+    ),
+    vec!["rs", "statefulsets"],
+    noop_complete,
+    |matches, env, writer| {
+        let regex = match ::table::get_regex(&matches) {
+            Ok(r) => r,
+            Err(s) => {
+                write!(stderr(), "{}\n", s).unwrap_or(());
+                return;
+            }
+        };
+
+        let urlstr = if let Some(ref ns) = env.namespace {
+            format!("/apis/apps/v1/namespaces/{}/statefulsets", ns)
```

Comment:
does the `/apis/apps/v1beta1/namespaces/{}/statefulsets/{}` url work for the newer versions you're using?  some of the clusters I manage are still on older versions and this url doesn't work.  if the `v1beta1` url works for you, let's use that since it'll support more things.

at some point we'll need to detect what version of k8s the server is running and switch based on that, but that's obviously a bigger change.

Reply:
Hm, I've just tested it and it works fine (as it should be). It didn't before so I probably messed up somewhere else. 

Either way I'm changing the url to use `v1beta1`.